projects/rocprofiler-sdk/source/lib/rocprofiler-sdk/pc_sampling/ioctl/ioctl_adapter.cpp

```diff
@@ -48,6 +48,9 @@ namespace
 
 #define PC_SAMPLING_IOCTL_COMPUTE_VERSION(major, minor) ROCPROFILER_COMPUTE_VERSION(major, minor, 0)
 
+#define MINIMUM_PC_SAMPLING_MEC_FW_VERSION uint32_t(0x0000001a)
```

Comment:
We received a complain recently that our documentation might have stating the outdate FW. Here is the issue: #1140. After sync up with Jay, I think the minimal version should be 185 (b9). Please make the doc update to reflect this as well. Also, please state that this is true for both MI300A and MI300X, as I think the #1140 is about the MI300A. 

```suggestion
#define MINIMUM_PC_SAMPLING_MEC_FW_VERSION uint32_t(0x000000b9)
```

Reply:
Hi Vladimir, bumped the version number to the correct version - this time it mentions both MI300A and MI300X for stochastic pc-sampling